.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: build
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
this one just for build/test

Reply:
No, shouldn't impact them. it's just easier because then I can use bash scripts on the runner